### DISCUSSION TITLE
2004 somewhat damaged tape

### DISCUSSION THREAD

**Discussion by ThrivertonVHS** [ntsc-rs/ntsc-rs#149] [Presets]
First preset, just messed around and tried to make a damaged tape based off a 2004 tape I found on youtube
Hope yall enjoy!
![image](https://github.com/user-attachments/assets/0b31e7b7-30cc-42bd-93b5-7ef677bc6670)

[thrivervhsVersion1.json](https://github.com/user-attachments/files/16500835/thrivervhsVersion1.json)

**Comment by smfthe1199** [comment]
How is this damaged?

**Reply by Degamisu** [reply]
It isn't crazy damaged, just somewhat damaged.

**Reply by smfthe1199** [reply]
Oh ok
